LayoutTests/imported/w3c/web-platform-tests/mediacapture-streams/MediaDevices-SecureContext-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL MediaDevices and SecureContext assert_false: MediaDeviceInfo is not exposed expected false got true
+PASS MediaDevices and SecureContext
```

Comment:
Did you check Firefox and Chrome? Does it align us with them? If so, the commit log should say so. If not, then I'm not sure we want to do this. Matching the behavior of other browser is more important than passing a WPT test or matching a spec.

Based on quick local testing, I suspect your proposed change would make our behavior differ from shipping Chrome.

Reply:
Here - you can see from dashboard, we are only browser failing this test - https://wpt.fyi/results/mediacapture-streams/MediaDevices-SecureContext.html?label=experimental&label=master&aligned